subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -31,35 +31,30 @@ LOG_MODULE_REGISTER(net_wifi_shell, LOG_LEVEL_INF);
 #include "net_shell_private.h"
 #ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
 static const char ca_cert_test[] = {
-	#include <wifi_enterprise_test_certs/ca.pem.inc>
+#include <wifi_enterprise_test_certs/ca.pem.inc>
 };
 
 static const char client_cert_test[] = {
-	#include <wifi_enterprise_test_certs/client.pem.inc>
+#include <wifi_enterprise_test_certs/client.pem.inc>
 };
 
 static const char client_key_test[] = {
-	#include <wifi_enterprise_test_certs/client-key.pem.inc>
+#include <wifi_enterprise_test_certs/client-key.pem.inc>
 };
 #endif
 
 #define WIFI_SHELL_MODULE "wifi"
 
-#define WIFI_SHELL_MGMT_EVENTS_COMMON (NET_EVENT_WIFI_SCAN_DONE   |\
-				NET_EVENT_WIFI_CONNECT_RESULT     |\
-				NET_EVENT_WIFI_DISCONNECT_RESULT  |\
-				NET_EVENT_WIFI_TWT                |\
-				NET_EVENT_WIFI_RAW_SCAN_RESULT    |\
-				NET_EVENT_WIFI_AP_ENABLE_RESULT   |\
-				NET_EVENT_WIFI_AP_DISABLE_RESULT  |\
-				NET_EVENT_WIFI_AP_STA_CONNECTED   |\
-				NET_EVENT_WIFI_AP_STA_DISCONNECTED)
+#define WIFI_SHELL_MGMT_EVENTS_COMMON                                                              \
+	(NET_EVENT_WIFI_SCAN_DONE | NET_EVENT_WIFI_CONNECT_RESULT |                                \
+	 NET_EVENT_WIFI_DISCONNECT_RESULT | NET_EVENT_WIFI_TWT | NET_EVENT_WIFI_RAW_SCAN_RESULT |  \
+	 NET_EVENT_WIFI_AP_ENABLE_RESULT | NET_EVENT_WIFI_AP_DISABLE_RESULT |                      \
+	 NET_EVENT_WIFI_AP_STA_CONNECTED | NET_EVENT_WIFI_AP_STA_DISCONNECTED)
```

Comment:
I am not sure we should blindly do what the clang-format suggests. For example here the original code is much more readable.